runtime/src/fips.rs

```diff
@@ -0,0 +1,26 @@
+// Licensed under the Apache-2.0 license
+use caliptra_common::cprintln;
+use caliptra_drivers::CaliptraError;
+use caliptra_drivers::CaliptraResult;
+use caliptra_registers::mbox::enums::MboxStatusE;
+
+use crate::Drivers;
+
+pub struct FipsModule;
+
+impl FipsModule {
+    pub fn status(_env: &Drivers) -> CaliptraResult<MboxStatusE> {
```

Comment:
Can we change this to Version similar to the command name?

Reply:
Sure.

Reply:
Done